tests/migration_test.go

```diff
@@ -2784,58 +2759,37 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 
 			Context("[Serial]Should trigger event if the nodes doesn't contain MigrationSelectorLabel for the vmi host-model type", func() {
 
-				var originalNodeLabels []map[string]string
-				var originalNodeAnnotations []map[string]string
 				var vmi *v1.VirtualMachineInstance
-				var backupNodes []k8sv1.Node
+				var nodes []k8sv1.Node
 
 				BeforeEach(func() {
-					backupNodes = libnode.GetAllSchedulableNodes(virtClient).Items
-					if len(backupNodes) == 1 || len(backupNodes) > 10 {
-						Skip("This test can't run with single node and it's too slow to run with more than 10 nodes")
-					}
-					originalNodeLabels = make([]map[string]string, len(backupNodes))
-					originalNodeAnnotations = make([]map[string]string, len(backupNodes))
+					nodes = libnode.GetAllSchedulableNodes(virtClient).Items
+
 					By("Creating a VMI with default CPU mode")
 					vmi = alpineVMIWithEvictionStrategy()
 					vmi.Spec.Domain.CPU = &v1.CPU{Model: v1.CPUModeHostModel}
 
 					By("Starting the VirtualMachineInstance")
 					vmi = runVMIAndExpectLaunch(vmi, 240)
 
-					for indx, node := range backupNodes {
-						originalNodeLabels[indx] = copyMap(node.Labels)
-						originalNodeAnnotations[indx] = copyMap(node.Annotations)
-						disableNodeLabeller(&node, virtClient)
+					for indx, node := range nodes {
+						patchedNode := stopNodeLabeller(node.Name, virtClient)
+						Expect(patchedNode).ToNot(BeNil())
+						nodes[indx] = *patchedNode
 					}
 				})
 
 				AfterEach(func() {
 					By("Restore node to its original state")
-					for indx, node := range backupNodes {
-						currNode, err := virtClient.CoreV1().Nodes().Get(context.Background(), node.Name, metav1.GetOptions{})
-						Expect(err).ShouldNot(HaveOccurred())
-						currNode.Labels = originalNodeLabels[indx]
-						currNode.Annotations = originalNodeAnnotations[indx]
-						_, err = virtClient.CoreV1().Nodes().Update(context.Background(), currNode, metav1.UpdateOptions{})
-						Expect(err).ShouldNot(HaveOccurred())
-					}
-					for indx, node := range backupNodes {
-						var currNode *k8sv1.Node
-						Eventually(func() map[string]string {
-							currNode, err = virtClient.CoreV1().Nodes().Get(context.Background(), node.Name, metav1.GetOptions{})
-							Expect(err).ShouldNot(HaveOccurred())
-							return currNode.Labels
-						}, 10*time.Second, 1*time.Second).Should(Equal(originalNodeLabels[indx]), "Node should have original labels")
-						Expect(currNode.Annotations).To(Equal(originalNodeAnnotations[indx]))
+					for _, node := range nodes {
+						resumeNodeLabeller(node.Name, virtClient)
 					}
```

Comment:
Can you verify that labels with `v1.SupportedHostModelMigrationCPU` are in the node?